src/pykeen/metrics/ranking.py

```diff
@@ -44,6 +45,16 @@ def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = Non
         """
         raise NotImplementedError
 
+    def _yield_expected_values(
```

Comment:
The name is not really matching anymore: it does not yield expected values, but rather yields sampled values

Reply:
9054246